src/main/java/core/basesyntax/dao/ManufacturerJdbcDaoImpl.java

```diff
@@ -15,25 +15,21 @@
 
 @Dao
 public class ManufacturerJdbcDaoImpl implements ManufacturerDao {
-    private static final int INDEX_ONE = 1;
-    private static final int INDEX_TWO = 2;
-    private static final int INDEX_THREE = 3;
 
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
         String query = "INSERT INTO manufacturers (manufacturer_name, "
                 + "manufacturer_country) VALUE (?,?)";
-        try (Connection connection = ConnectionUtil.getConnection()) {
-            PreparedStatement createdStatement = connection
-                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
-            createdStatement.setString(INDEX_ONE, manufacturer.getName());
-            createdStatement.setString(INDEX_TWO, manufacturer.getCountry());
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createdStatement = connection
+                        .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            createdStatement.setString(1, manufacturer.getName());
+            createdStatement.setString(2, manufacturer.getCountry());
             createdStatement.execute();
             ResultSet resultSet = createdStatement.getGeneratedKeys();
             if (resultSet.next()) {
-                manufacturer.setId((Long)resultSet.getObject(INDEX_ONE));
+                manufacturer.setId(resultSet.getLong(1));
```

Comment:
be attentive with checklist - https://mate-academy.github.io/jv-program-fulltime/02_jdbc/homework/05_JDBC_HW.html

Reply:
yes, I saw this. Sorry, totally forgot that we can return Object of Long.class